app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/completeregistration/CompleteRegistrationNavigation.kt

```diff
@@ -63,12 +63,23 @@ fun NavGraphBuilder.completeRegistrationDestination(
             navArgument(VERIFICATION_TOKEN) { type = NavType.StringType },
             navArgument(FROM_EMAIL) { type = NavType.BoolType },
         ),
-    ) {
+    ) { navBackStackEntry ->
+        val generatedPassword = navBackStackEntry
+            .savedStateHandle
+            .get<String>(CompleteRegistrationNavigation.GENERATED_PASSWORD)
```

Comment:
The `savedStateHandle` is already inside the VM, can you not just get it from there?

Reply:
Did you try observing the saveStateHandle?

```kotlin
        savedStateHandle
            .getStateFlow(key = CompleteRegistrationNavigation.GENERATED_PASSWORD, "")
            .map { VaultAction.Internal.DoTheThing(it) }
            .onEach(::sendAction)
            .launchIn(viewModelScope)
```

Reply:
TIL, will give it the ol college try

Reply:
for posterity: it did not work.